pkg/cloud/vsphere/context/cluster_context.go

```diff
@@ -102,7 +102,7 @@ func NewClusterContext(params *ClusterContextParams) (*ClusterContext, error) {
 	if logr == nil {
 		logr = klogr.New().WithName("default-logger")
 	}
-	logr = logr.WithName(params.Cluster.APIVersion).WithName(params.Cluster.Namespace).WithName(params.Cluster.Name)
+	logr = logr.WithName(params.Cluster.Name)
```

Comment:
The current model is in sync with CAPA and provides necessary information. What is the reason for changing it?

Reply:
Adding the full qualified API version extends the line length significantly, leaving less room real estate for meaningful log data

Reply:
I understand your concern, but because logs could be interspersed from multiple incoming calls, having the context of the object being logged is important as log data is flushed in real-time. If you still wish to make this change, please bring it up in the next upstream call. For now let's leave it as is so we can merge the rest of the PR : )

Reply:
👍

Reply:
Hi @moshloop,

FWIW, we literally just got done demoing 0.3.0 where the logs verbosity helped. A prior machine set was still being deleted during the creation of a new one, and log entries were being interlaced with each other.